pkg/mcproducer/scripts/replace-kernel-module.sh

```diff
@@ -31,14 +32,17 @@ if [ -n "$(podman images -q $full_kernel_module_image 2> /dev/null)" ]; then
     podman volume create $worker_volume_name
     podman pod create --name $worker_pod_name
     echo "creating init container"
+    copycmd="mkdir -p /tmp/opt/lib/modules && cp -R /opt/lib/modules/* /tmp/opt/lib/modules;"
+    if [[ -n "$FIRMWARE_FILES_PATH" ]]; then
+      copycmd+=" mkdir -p /tmp${firmware_files_path} && cp -R ${firmware_files_path}/* /tmp${firmware_files_path}"
```

Comment:
This won't work if the firmware path doesn't start with `/`

Reply:
Maybe the accepted solution here can be of help
https://unix.stackexchange.com/questions/23208/building-paths-robustly

Reply:
Based on the accepted solution and on what we do in kmm, I updated it to be:
`    if [[ -n "$FIRMWARE_FILES_PATH" ]]; then
      folders=("tmp" "$firmware_files_path");
      path_to_copy_firmware=$(printf '/%s' "${folders[@]%/}")
      copycmd+=" mkdir -p ${path_to_copy_firmware} && \
      cp -R ${firmware_files_path}/* ${path_to_copy_firmware}"
    fi
    `
    In kmm we also use filepath.Join function to `*`, but here I think this approach would be more clean this way.